matrix_client/api.py

```diff
@@ -621,6 +691,41 @@ def get_text_body(self, text, msgtype="m.text"):
             "body": text
         }
 
+    def get_call_invite_content(self, call_id, sdp, version, lifetime, types="offer"):
```

Comment:
Why is this dictionary constructed under a separate method rather than just in the method body of `send_call_invite`?

Stylistically, if kept as a separate method, `types` kwarg should probably be `type`.